lib/generators/rbui/install/templates/application.tailwind.css.tt

```diff
@@ -0,0 +1,60 @@
+@tailwind base;
```

Comment:
you should use the variables on this file: https://github.com/PhlexUI/PhlexUI-web/blob/v1/app/assets/stylesheets/application.tailwind.css